src/igvfd/schemas/sample_ontology_term.json

```diff
@@ -0,0 +1,64 @@
+{
+    "title": "Sample ontology term",
+    "description": "Schema for an ontology term.",
+    "$id": "/profiles/sample_ontology_term.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "term_id",
+        "term_name"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        },
+        {
+            "$ref": "mixins.json#/aliases"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        },
+        {
+            "$ref": "mixins.json#/notes"
+        },
+        {
+            "$ref": "ontology_term.json#/properties"
```

Comment:
Sorry for my earlier comment.  I was not seeing this line as I was looking at the top of the list.